README.md

```diff
@@ -105,8 +105,8 @@ Templates give you full control of each component of your application.
 Sometimes your application is simple enough and you don't want to bother with templates. In that case, you can let OpenShift inspect your source code and create the required components automatically for you:
 
 ```bash
-$ oc new-app openshift/python-33-centos7~https://github.com/openshift/django-ex
-imageStreams/python-33-centos7
+$ oc new-app openshift/python-34-centos7~https://github.com/openshift/django-ex
+imageStreams/python-34-centos7
```

Comment:
the correct 3.4 image name is centos/python-34-centos7, the openshift one is deprecated.